lib/gui/app/models/local-settings.js

```diff
@@ -38,10 +48,19 @@ const settingsStorage = new Storage(LOCAL_STORAGE_SETTINGS_KEY)
  */
 const RCFILE = '.etcher.json'
 
+/**
+ * @summary Configuration file path
+ * @type {String}
+ * @constant
+ */
+const HOME_CONFIG_PATH = process.platform === 'win32'
+  ? path.join(USER_DATA_DIR, RCFILE)
```

Comment:
Is this special-casing of the filename still relevant on win32 now, given that `USER_DATA_DIR` will always end with `etcher` ?

Reply:
Ah, good catch – this is obsolete now, as is `RCFILE`